snow/networking/handler/handler.go

```diff
@@ -130,6 +132,7 @@ type handler struct {
 // Initialize this consensus handler
 // [engine] must be initialized before initializing this handler
 func New(
+	haltBootstrapping func(),
```

Comment:
nit: should we define this as the last argument or after `ctx`, while it's not `context.Context`, still it seems that `ctx` is passed around as the first argument

Reply:
Done